CMSgov/cms-carts-seds
# [CMDCT-3810] Remove the unused deps in uisrc

### Description



### Related ticket(s)


CMDCT-3810

---
### How to test

run yarn in services/ui-src folder and see that there are no dependency errors
Run the application and edit information, submit, swap to an admin, and confirm you see your changes as an admin.

### Notes


**Dependencies Removed**
React-searchable-dropdown
react-papaparse
react-number-format
react-json-view
react-json-editor-viewer
react-json-editor-ajrm
query-string
fortawesome/free-regular-svg-icons, 
 
**devDependencies Removed**
eslint-config-react-app

---
### Pre-review checklist


---
### Pre-merge checklist


#### Review
